src/ProjectService.Business/Commands/Project/EditProjectCommand.cs

```diff
@@ -93,21 +96,23 @@ public OperationResultResponse<bool> Execute(Guid projectId, JsonPatchDocument<E
 
             GetProjectFilter filter = new GetProjectFilter { ProjectId = projectId };
```

Comment:
remove this variable, and replace creating filter to 99 line, at the time of calling the method

Reply:
and why don't you use the enable user flag, so just enable it, and after that you won't need to make an extra request to the database

Reply:
done